src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -0,0 +1,29 @@
+import {
+	Component,
+	HostBinding,
+	Input
+} from "@angular/core";
+
+import { Observable, of } from "rxjs";
+
+@Component({
+	selector: "ibm-breadcrumb-item",
+	template: `
+	<a class="bx--link"
+		href="{{href}}"
+		*ngIf="href; else content">
+		<ng-container *ngTemplateOutlet="content"></ng-container>
+	</a>
+	<ng-template #content>
```

Comment:
if you exposed this through a property, you could pass this into the dynamic component though an input. Then you can keep the same usage for dynamic and non-dynamic breadcrumbs.

Reply:
@cal-smith I haven't been able to get the text content for `<ng-template #content>`

I tried adding this property `@ViewChild('content', {read: ElementRef}) content: ElementRef;`

When I try `this.content.nativeElement.textContent` it is en empty string.

Reply:
@cal-smith @zvonimirfras I'm still not clear on what changes you want made here. All other feedback has been addressed.

Reply:
So if you get the `ng-template` TemplateRef with a `ViewChild` (or `ContentChild`?), and change the `Input` to take a `TemplateRef` you can pass down the whole template instead of just the text.

so instead of  `<ng-container *ngTemplateOutlet="content"></ng-container>` you'd want something like `<ng-container *ngTemplateOutlet="getContentTemplate()"></ng-container>` where `getContentTemplate` does something like:

```typescript
@ViewChild('content') template: TemplateRef;
@Input() contentTemplate: TemplateRef; // replaces @Input() text

public getContentTemplate() : TemplateRef {
    if (this.contentTemplate) {
        return this.contentTemplate;
    }
    return this.template;
}
```

Reply:
Basically let the user pass in a whole template, but by default use our template that wraps `ng-content`

Reply:
@cal-smith I've tried that but the net result is nothing get's displayed inside the breadcrumb items